tests/defer/models.py

```diff
@@ -5,6 +5,19 @@
 from django.db import models
 
 
+class ParentModel(models.Model):
```

Comment:
I think you could edit some existing models rather than adding three new models. We try to avoid adding new models since it decreases the runtime of the tests.